blankly/exchanges/interfaces/ftx_futures/ftx_futures_interface.py

```diff
@@ -326,13 +327,14 @@ def get_funding_rate_resolution(self) -> int:
         return time_builder.build_hour()
 
     def get_product_history(self, symbol, epoch_start, epoch_stop, resolution):
-        raise NotImplementedError
+        # reuse impl from SPOT interface, it's the same thing
+        return FTXInterface.get_product_history(symbol, epoch_start, epoch_stop, resolution)
```

Comment:
I think we talked about it on discord but this is the one spot that might be sketchy